spec/migrations/20150921204114_add_vmware_ro_datastores_to_hosts_storages_spec.rb

```diff
@@ -2,11 +2,38 @@
 require_migration
 
 describe AddVmwareRoDatastoresToHostsStorages do
+  class AddVmwareRoDatastoresToHostsStorages::HostsStorage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    self.table_name = "hosts_storages"
+  end
+
+  let(:host_storages_stub) { migration_stub(:HostStorage) }
   let(:hosts_storages_stub) { migration_stub(:HostsStorage) }
 
   migration_context :up do
     it "Adds ID in correct region" do
+      seq_start = ActiveRecord::Base.rails_sequence_start
+      seq_start = 1 if seq_start == 0
+
+      h = hosts_storages_stub.create(:host_id => 1, :storage_id => 2)
+      migrate
+      h = host_storages_stub.first
+
+      expect(h.id).to eq(seq_start)
```

Comment:
This is probably not right if run with other specs...the sequence keeps moving in the database between tests, so you shouldn't rely on that.  A better test might be to determine the REGION of the current system (MiqRegion.my_region_number), and verify that the id is in the correct region range (there's another region helper method that can do that for you ...I think it's `in_current_region?` or something like that.)